common/terms.html

```diff
@@ -136,6 +136,10 @@
     <dd>
       <p>Usable by users in ways they can control. References in this document relate to <cite><a href="https://www.w3.org/TR/WCAG21/#operable"><abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.1 Principle 2: Content must be operable</a></cite> [[WCAG21]]. See <a>Keyboard Accessible</a>.</p>
     </dd>
+    <dt><dfn data-lt="owned child|child element|child|children|child elements">Owned Child</dfn></dt>
+    <dd>
+      <p>An 'owned child' is a direct <abbr title="Document Object Model">DOM</abbr> child of the <a>element</a>, a descendant of the <a>element</a> with only <rref>generic</rref> or <rref>presentation</rref> elements intervening, or a child specified via an <pref>aria-owns</pref> relationship to the element.</p>
```

Comment:
If an intervening generic aria-owns something, is it an owned child?

Reply:
No, it would be an owned descendant, but not an owned child.